.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -32,15 +32,33 @@ labels: 'Status: Triage'
 - [ ] `build/tooling/chore`
 - [ ] other:
 
-### Version:
+### Dependencies + versions
```

Comment:
Thanks @dimkl :)

We can probably discuss adding a Clerk.info() or similar API that'd copy all these debug data in markdown format to make our users' life easier. I think Bun does this extremely well